pkg/metrics/otel.go

```diff
@@ -54,13 +54,18 @@ exporters:
   googlecloud/kubernetes:
     metric:
       prefix: "kubernetes.io/internal/addons/config_sync/"
-      skip_create_descriptor: false
+      skip_create_descriptor: true
+      instrumentation_library_labels: false
+      create_service_timeseries: true
```

Comment:
Why do we need to set this field to true?

Reply:
Confirmed that this is a recommended configuration for 'system metrics' that starts with `kubernetes.io/`. It uses CreateTimeSeries API and has its own quota so the writes to custom API from user won't break this ingestion pipeline. 

From the [exporter doc](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter#configuration-reference) it also auto skips creating metric descriptor, but it's still worthy to list `skip_create_descriptor` out to reduce confusion.

Reply:
Add a comment to explain this in the file.

Reply:
Done!